Integrations/integration-ESM-v10.yml

```diff
@@ -0,0 +1,460 @@
+
+            query = ''
+            if (assigned_user == 'ME'):
+                user = self.get_user()
+                query = json.dumps({ 'assignedUser' : user})
+            elif assigned_user:
+                query = json.dumps({ 'assignedUser' : json.loads(assigned_user)})
+
+            res = self.cmdquery(cmd, query)
+
+            for alarm in res:
+                alarm['ID'] = alarm['id']["value"]
+                del alarm["id"]
+
+            return res
+
+    def alarms_to_entry(alarms):
+        if not alarms:
+                return "No alarms were found"
+
+        context = { 'Alarm(val.ID && val.ID == obj.ID)': alarms }
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': alarms,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tblToMd('Result:' ,alarms),
+            'EntryContext': context
+        }
+
+    def alarms_to_incidents(alarms):
+        incidents = []
+        for alarm in alarms:
+            incidents.append({
+                'name': alarm['summary'],
+                'details': 'Alarm {} , ID : {} , was triggered by condition type: {}'.format(alarm['alarmName'], alarm['ID'], alarm['conditionType']),
+                'severity': severity_to_level(alarm['severity'])
+            })
+        return incidents
+
+    esm = NitroESM(demisto.params()['ip']+':'+demisto.params()['port'], demisto.params()['username'], demisto.params()['password'])
+
+    if demisto.command() == 'test-module':
+        esm.login()
+        demisto.results('ok')
+        sys.exit(0)
+    if demisto.command() == 'esm-fetch-fields':
+        res = esm.featchAllFields()
+        demisto.results(res)
+        sys.exit(0)
+    if demisto.command() == 'esm-search':
+        args = demisto.args()
+        res = esm.search(
+            demisto.get(args, 'timeRange'),
+            demisto.get(args, 'customStart'),
+            demisto.get(args, 'customEnd'),
+            json.loads(demisto.get(args, 'filters')),
+            demisto.get(args, 'queryType') or 'EVENT',
+            demisto.get(args, 'maxWait') or 30
+            )
+        demisto.results(res)
+        sys.exit(0)
+    if demisto.command() == 'esm-fetch-alarms':
+        args= demisto.args()
+        res = esm.fetch_alarms(
+            demisto.get(args, 'timeRange'),
+            demisto.get(args, 'customStart'),
+            demisto.get(args, 'customEnd'),
+            demisto.get(args, 'assignedUser')
+        )
+
+        entry = alarms_to_entry(res)
+
+        demisto.results(entry)
+        sys.exit(0)
+    if demisto.command() == 'fetch-incidents':
+
+        last_run = demisto.getLastRun()
+        start = last_run and last_run['value']
+
+        if (not start):
+            start = (datetime.datetime.now() - datetime.timedelta(minutes=1)).isoformat()
+
+        end = datetime.datetime.now().isoformat()
```

Comment:
@meirwah will check that against our instance
I noticed that in go implementation we use zulu time..

why shouldn't we do this here also?